coderd/database/queries/workspaceagents.sql

```diff
@@ -299,4 +299,14 @@ INSERT INTO
         status
     )
 VALUES
-    ($1, $2, $3, $4, $5, $6);
+    ($1, $2, $3, $4, $5, $6)
+RETURNING workspace_agent_script_timings.*;
+
+-- name: GetWorkspaceAgentScriptTimingsByWorkspaceID :many
```

Comment:
I think this needs to be queried by workspace build ID, otherwise it'll return timings across multiple builds, no?

Reply:
100% Correct 🤦 I don't know why I didn't catch that.

Reply:
Has this been addressed yet?

Reply:
Nvm, was looking at a stale commit.